--- vnc2swf-0.5.0/Makefile.in.orig	2005-10-29 07:22:05.000000000 +0200
+++ vnc2swf-0.5.0/Makefile.in	2013-05-11 07:58:55.173400289 +0200
@@ -45,7 +45,7 @@
 
 DEP_LIBS = rfb/librfb.a rdr/librdr.a
 
-EXTRA_LIBS = @X_PRE_LIBS@ @X_LIBS@ -lXmu -lXaw -lXt -lX11 -lXext @X_EXTRA_LIBS@
+EXTRA_LIBS = @X_LIBS@ -lXmu -lXaw -lXt -lX11 -lXext @X_EXTRA_LIBS@
 
 
 DIR_CPPFLAGS = -I$(top_srcdir) @SOCKLEN_T_DEFINE@ @X_CFLAGS@ # X_CFLAGS are really CPPFLAGS
